### ISSUE TITLE
BiasT and TR\RX with the same antenna

### ISSUE THREAD

**Issue by nir4x6fk** [plutoplus/plutoplus#15]
Hello,
I'm planning to have a Pluto+
* Do the Pluto+ have a BiasT?
*  there is the possibility to connect one antenna to TX and Rx port?
Thank's
Nir  4X6FK

**Comment by fei666888** [comment]
No Bias-T on pluto+，you can add one outside

**Comment by nir4x6fk** [comment]
Thank's for the info.
For TX and Rx I must use 2 separate antennas?

**Comment by fei666888** [comment]
> 
Yes, this is the same as ADI’s plutosdr, it needs an external switch to share an antenna, or use a duplexer
